### PR TITLE
Second round of refactorings

### PR DESCRIPTION
**Pull request by VoyTechnology**
Not yet finished. I am moving them a bunch at a time for easier review.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ New admins, read:
 
 *  [Pygmalion](pyg)
 
-*  `<del>`[Carbon](carbon)`</del>`
+*  `<del>`[Carbon](legacy/machines/carbon)`</del>`
```
Comment (butlerx) [inline docs/index.md]: use ~~ stiketrough as thats what <del> is for

File: docs/index.md [hunk 2]
```diff
@@ -224,18 +224,18 @@ Stuff from the Ubuntu 6.06 days
 
 *  [basementcat ](basementcat)
 
-*  `<del>` [ceilingcat](ceilingcat)`</del>`
+*  `<del>` [ceilingcat](legacy/machines/ceilingcat)`</del>`
```
Comment (butlerx) [inline docs/index.md]: same as appove

File: docs/index.md [hunk 3]
```diff
@@ -349,9 +349,9 @@ This stuff doesn't reflect the current setup, but may be useful sometime.
 
 # System Documentation Redundancy
```
Comment (butlerx) [inline docs/index.md]: tbh this whole section is legacy, and half of these pages are gone and the one that isnt is quite broken

Comment (butlerx) [inline docs/index.md]: also we dont host docs anymore and are hosted on github redentenly

File: docs/index.md (hunk 1 above)
Comment (butlerx) [inline docs/index.md]: or remove the `` around the <del>

Comment (VoyTechnology) [inline docs/index.md]: This PR is simply moving the files around and fixes for links. I will fix formatting problems in a separate PR.

File: docs/index.md (hunk 2 above)
Comment (VoyTechnology) [inline docs/index.md]: Also the same reply :)

File: docs/index.md (hunk 3 above)
Comment (VoyTechnology) [inline docs/index.md]: This change was simply to fix the links that existed. I will remove dead links in another PR.

Comment (butlerx) [inline docs/index.md]: >This PR is simply moving the files around and fixes for links
if we move files all links to those files should be fixed

File: docs/index.md (hunk 1 above)
Comment (CSmartt) [inline docs/index.md]: Remove backticks or use ~, seeing as you're already modifying this line.

File: docs/index.md [hunk 4]
```diff
@@ -181,28 +181,28 @@ Stuff from the Ubuntu 6.06 days
...
-*  [Installing Debian on Cynic](cynicdebian)
+*  [Installing Debian on Cynic](legacy/procedures/cynicdebian)
 ####  Daniel - VM Host
 
-*  [daniel](daniel)
+*  [daniel](legacy/machines/daniel)
 
 *  [ x4140 ILOM](x4140ilom )
 
 *  [ VM info and password resets](rbvmctl )
```
Comment (CSmartt) [inline docs/index.md]: Legacy

File: docs/index.md [hunk 5]
```diff
@@ -181,28 +181,28 @@ Stuff from the Ubuntu 6.06 days
...
 
-*  [ Compiling Xen](compilingxen )
+*  [ Compiling Xen](procedures/compilingxen )
 
 *  [RBVM management tool](rbvm)
```
Comment (CSmartt) [inline docs/index.md]: Also legacy

File: docs/index.md [hunk 6]
```diff
@@ -224,18 +224,18 @@ Stuff from the Ubuntu 6.06 days
 
 *  [basementcat ](basementcat)
```
Comment (CSmartt) [inline docs/index.md]: Legacy, basementcat was stolen from storage a few months ago.

File: docs/index.md [hunk 7]
```diff
@@ -224,18 +224,18 @@ Stuff from the Ubuntu 6.06 days
...
 ## Hardware
 
 ### Switches
 
-*  [Cisco IOS](ciscoios) 
+*  [Cisco IOS](ciscoios)
 
 *  [hadron](hadron)
 
 *  `<del>`[enzyme](enzyme)`</del>`
```
Comment (CSmartt) [inline docs/index.md]: Backticks/~ again. Also legacy.

File: docs/index.md [hunk 8]
```diff
@@ -319,13 +319,13 @@ This stuff doesn't reflect the current setup, but may be useful sometime.
 
 *  [Open Ports](openports)
 
-*  [IRC proxy](ircproxy) 
+*  [IRC proxy](ircproxy)
 
-*  [Apache configuration on Solaris 10](apachesolaris)
+*  [Apache configuration on Solaris 10](legacy/procedures/apachesolaris)
 
 *  [PHP configuration on Solaris 10](phpsolaris)
```
Comment (CSmartt) [inline docs/index.md]: All Solaris can be considered legacy.

File: docs/index.md (hunk 4 above)
Comment (VoyTechnology) [inline docs/index.md]: I am doing the moves in waves so its easier to review. Will move this to legacy in another round.

File: docs/index.md (hunk 5 above)
Comment (VoyTechnology) [inline docs/index.md]: As above

File: docs/index.md (hunk 8 above)
Comment (VoyTechnology) [inline docs/index.md]: I have not reached that far yet in the refactoring. I am doing it in waves

File: docs/index.md (hunk 4 above)
Comment (butlerx) [inline docs/index.md]: i think we would rather havew one move rather then waves

File: mkdocs.yml [hunk 9]
```diff
@@ -7,44 +7,45 @@ pages:
    - Machines:
```
Comment (butlerx) [inline mkdocs.yml]: mov legAcy to the bottom

File: docs/index.md (hunk 1 above)
Comment (VoyTechnology) [inline docs/index.md]: https://github.com/redbrick/docs/pull/15

**Comment by VoyTechnology** [comment]
Closing as #19 does fixes the things requested here

**Pull request closed without merging**